runtime/onert/core/include/interp/InterpExecutor.h

```diff
@@ -59,9 +59,12 @@ class InterpExecutor final : public exec::IExecutor
    */
   void execute(const exec::IODescription &desc) final;
 
+  void fillOutputShapes(std::unordered_map<ir::IOIndex, ir::Shape> *output_shapes) override;
+
 private:
   const ir::Graph &_graph;
   ir::OperandIndexMap<std::shared_ptr<ITensor>> _tensor_map;
+  bool _execution_done;
```

Comment:
`InterpExecutor` is thread-unsafe, so it should not have state variable. 
One solution is to use `IODescription`, or pass by `execute(..)`'s parameter to fill shape result.

Reply:
Thanks for the comment.
I think I'd better to use `IODescription` for both `InterpExecutor` and `ExecutorBase`.
I will re-write the code. Thanks!